src/platform/i386/isr.h

```diff
@@ -6,11 +6,13 @@
 #include "chal_asm_inc.h"
 
 enum {
-	IRQ_PAGE_FAULT = 14,
-	IRQ_PIT        = 32,
-	IRQ_KEYBOARD   = 33,
-	IRQ_SERIAL     = 36
+	IRQ_DOUBLE_FAULT = 8,
+	IRQ_PAGE_FAULT   = 14,
+	IRQ_PIT          = 32,
```

Comment:
You'll find that Jakob had to alter this as well.  Please make sure your changes are consistent with his.